src/test/java/org/cactoos/collection/FilteredTest.java

```diff
@@ -54,6 +54,7 @@ public void behavesAsCollection() throws Exception {
     @Test
     public void filterList() {
         MatcherAssert.assertThat(
+            "Length must change",
```

Comment:
`new Assertion<>().affirm()`

Reply:
@umbrah message should be "length equal to number of items matching filter"

Reply:
@umbrah actually I don't understand what the difference is between this test and the `size()` test below. I think this one can be deleted